<!DOCTYPE html>
<html lang="en">

<!-- head of html section starts-->
<head>
  <meta charset="UTF-8">
  <meta name="viewport" content="width=device-width, initial-scale=1.0">
   <title>phylogenize2</title>
   <link rel="stylesheet" href="style.css">
</head>

<!-- body of html section starts-->
<body>
    <!--header of body section starts-->

  <header class="hero">
    <!--add a figure about logic of phylogenize2/PGLS-->
        <img src="assets/phylogenize2/background_phylo.png" alt="Project banner">
        <h1><em>phylogenize2</em></h1>
        <p> A robust phylogenetic regression framework for microbiome analysis</p>
        <p> Nghi (Nia) Tran, Kathyrn Kananen, Patrick Bradley</p>
  </header>

    <!--main of body section starts-->
<main>
    <section>
        <h1>Project Overview</h1>
        <p> phylogenize is a R-based and conda package to utilize <strong>PGLS (phylogenetic generalized least square)</strong> to use gene contents to predict for environmental phenotypes
        (such as specificity, prevalence or abundance) directly from sequencing data, while also accounting for the evolutionary relationships between microbes.
        </p> 
    </section>

    <section>
        <h1>Methods</h1>
        <p> We use <strong>Phylogenetic Generalized Least Square</strong> (phylolm package), where predictor variables are gene presence/absence dataset (generated from UniRef, UHGP, and GTDB dataset clustering with Anivo's mmseq).
            and outcome variables are count data (processed with QC, BBDuk, Kraken/Bracken), which will be transformed into phenotype data (prevalence/specificity or abundance, depending on user's choice)</p>
        <p>To account for outlier and non-parametric datasets, we implemented <strong>phylogenetic independent contrast - PIC</strong> (R ape package), combined with regression with empirical null distribution estimation method <strong>("permulate")</strong>.
            This will help to reduce FPR and increase sensitivity. We use <strong>early-stopping</strong> (an adaptive machine-learning techniques to halt model when no further signficant hits are found) that remarkably save computing runtime
            and resources.</p>
    </section>

    <section>
        <h1>Results Summary</h1>

        <figure>
            <img src="assets/phylogenize2/runtime_compared.svg" alt="kkk">
            <figcaption>
                <strong>Figure 1</strong>: Application of our new, phylogenetic statistical approach - Permul - 
                shows a significant <em>decrease in computing resources</em>.
            </figcaption>   
        </figure>
        
        <figure>
            <img src="assets/phylogenize2/FDR.png">
            <figcaption>
                <strong>Figure 2</strong>: Permul approach in phylogenize2 indicates significantly <em>less false positives</em>
                compared to other previous methods
                
            </figcaption>   
        </figure>

        <figure>
            <img src="assets/phylogenize2/power.png">
            <figcaption>
                <strong>Figure 3</strong>:... while also yields a better<em>improved sensitivity </em>than others.
            </figcaption>   
        </figure>

        <figure>
            <img src="assets/phylogenize2/taraO.png">
            <figcaption>
                <strong>Figure 4</strong>: One of the significant protein hits identified from 
                our new statistical method, from the <em>marine microbiome (TaraOceans dataset)</em> that 
                increases in the mesopelagic zone compared to the surface zone. 
            </figcaption>   
        </figure>

        <figure>
            <img src="assets/phylogenize2/mice-gut.png">
            <figcaption>
                <strong>Figure 5</strong> A significant protein hits from the <em>mouse-gut microbiome</em> 
                that undergo intermittent fasting compared to normal diet using our new approach.
            </figcaption>   
        </figure>

        <figure>
            <img src="assets/phylogenize2/cirrhosis.png">
            <figcaption>
                <strong>Figure 6</strong> Permul implemented in phylogenize2 detects 2 member of protein family 
                that increase in <em>cirrhosis gut microbiome</em> 
            </figcaption>   
        </figure>

    <p><a href="index.html">Home</a>

    
</main>

</body>

</html>